lib/dialects/sqlite/data-types.js

```diff
@@ -23,6 +23,17 @@ module.exports = BaseTypes => {
   BaseTypes.REAL.types.sqlite = ['REAL'];
   BaseTypes.DOUBLE.types.sqlite = ['DOUBLE PRECISION'];
   BaseTypes.GEOMETRY.types.sqlite = false;
+  BaseTypes.JSON.types.sqlite = ['JSON', 'JSONB'];
+
+  function JSONTYPE() {
```

Comment:
We need `JSON.parse` so we can't rename function itself, but its okay I think. So no changes